crypto/fipsmodule/evp/evp.c

```diff
@@ -436,6 +447,10 @@ int EVP_PKEY_assign(EVP_PKEY *pkey, int type, void *key) {
       return EVP_PKEY_assign_EC_KEY(pkey, key);
     case EVP_PKEY_DH:
       return EVP_PKEY_assign_DH(pkey, key);
+#ifdef ENABLE_DILITHIUM
+    case EVP_PKEY_PQDSA:
+      return EVP_PKEY_assign_PQDSA_KEY(pkey, key);
```

Comment:
why can't you use the existing default `EVP_PKEY_set_type`?

Reply:
ahh, I see! fixed in [bcbb832](https://github.com/aws/aws-lc/pull/1963/commits/bcbb8322f367532268ad01c5a940a5518b349562)